### PR TITLE
Script Updates for Gradle wrapper

### PR DESCRIPTION
**Pull request by bhariharan**

### CODE REVIEW

**Comment by bhariharan** [comment]
@wmathurin

File: tools/postinstall-android.js [hunk 1]
```diff
@@ -93,8 +94,14 @@ shelljs.sed('-i', oldSalesforceSdkDep, 'compile project\(\':SalesforceSDK\'\)',
 shelljs.sed('-i', oldSmartStoreDep, 'compile project\(\':SmartStore\'\)', path.join(appProjectRoot, 'SmartSync', 'build.gradle'));
 
 console.log('Fixing root level Gradle file for the generated app');
-var oldEntry = "include \":CordovaLib\"";
-var newEntry = "include \":CordovaLib\"\ninclude \":SalesforceSDK\"\ninclude \":SmartStore\"\ninclude \":SmartSync\"";
-shelljs.sed('-i', oldEntry, newEntry, path.join(appProjectRoot, 'settings.gradle'));
+shelljs.echo("include \":SalesforceSDK\"\n").toEnd(path.join(appProjectRoot, 'settings.gradle'));
```
Comment (bhariharan) [inline tools/postinstall-android.js]: And a small optimization here - better than search and replace.

**Comment by wmathurin** [comment]
:+1:

**Comment by kchitalia** [comment]
:+1:

**Pull request merged**